pkg/csi/cinder/openstack/openstack.go

```diff
@@ -206,3 +214,13 @@ func GetOpenStackProvider() (IOpenStack, error) {
 
 	return OsInstance, nil
 }
+
+//GetMaxVolLimit returns max vol limit
+func GetMaxVolLimit() int64 {
+	if cfg.BlockStorage.NodeVolumeAttachLimit > 0 && cfg.BlockStorage.NodeVolumeAttachLimit <= 256 {
```

Comment:
doc update for this new parameter seems to be missing?

Reply:
yes, this need some doc to indicate the limit..

Reply:
others look good